ws-gateway-api/src/main/java/com/cosmin/wsgateway/api/Application.java

```diff
@@ -25,15 +25,14 @@
     @Autowired
     private GatewayProperties gatewayProperties;
 
-    public static void main(String[] args) {
-        Schedulers.enableMetrics();
+    public static void main(String[] args) { Schedulers.enableMetrics();
```

Comment:
**[checkstyle]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
'{' at column 44 should have line break after.